### ISSUE TITLE
Why are some legacy/deprecated entries explicitly excluded in ExtValidationKeySelect

### ISSUE THREAD

**Issue by kukel** [phax/peppol-practical#10]
There is a comment line in there that says they are explicitly removed, just not why and I cannot seem to find anything in the peppol bis3 release-notes. They seem to be all bis3 t* versions from the spring 2020 release. 

https://github.com/phax/peppol-practical/blob/54887adfa2c1e2cffc1e7e615a34b9bcc872d5c9/src/main/java/com/helger/peppol/phive/ExtValidationKeySelect.java#L36

	PeppolValidation3_10_0.VID_OPENPEPPOL_T01_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T16_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T19_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T58_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T71_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T76_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T77_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T110_V3,
	PeppolValidation3_10_0.VID_OPENPEPPOL_T111_V3

**Comment by phax** [comment]
That was the first release, where I switched from calling the VES IDs `eu.peppol.bis3:t01:3.10.0` to `eu.peppol.bis3:order:3.10.0` which is imho much more understandable.
Because version 3.10.0 registeres BOTH versions, I decided to explicitly remove the "duplicate" ones in Peppol Practical.

The old naming `t01` stems from the "Transaction 01" in the CEN BII documents. Since that knowledge can now be considered "ancient", the usage of `order` seems to be more approriate ;-)
